File: assessment_5/Conditionals.js
// Input: Get a number from the user
let number = prompt("Enter a number:");

// Convert the input to a number
number = Number(number);

// Conditional: Check if the number is positive, negative, or zero
if (number > 0) {
    console.log("The number is positive.");
} else if (number < 0) {
    console.log("The number is negative.");
} else {
    console.log("The number is zero.");
}

File: Conditionals.cpp
#include <iostream>
using namespace std;

int main() {
    int number;

    // Input: Get a number from the user
    cout << "Enter a number: ";
    cin >> number;

    // Conditional: Check if the number is positive, negative, or zero
    if (number > 0) {
        cout << "The number is positive." << endl;
    } else if (number < 0) {
        cout << "The number is negative." << endl;
    } else {
        cout << "The number is zero." << endl;
    }

    return 0;
}
